plugin/pkg/scheduler/api/types.go

```diff
@@ -92,14 +92,14 @@ type PriorityArgument struct {
 	LabelPreference *LabelPreference
 }
 
-// Holds the parameters that are used to configure the corresponding predicate
+// ServiceAffinity holds the parameters that are used to configure the corresponding predicate
 type ServiceAffinity struct {
 	// The list of labels that identify node "groups"
 	// All of the labels should match for the node to be considered a fit for hosting the pod
 	Labels []string
 }
 
-// Holds the parameters that are used to configure the corresponding predicate
+// LabelsPresence holds the parameters that are used to configure the corresponding predicate
```

Comment:
LabelsPresence holds the parameters that are used to configure the corresponding predicate in scheduler policy configuration.